app/frontend/src/utils/transformUtils.js

```diff
@@ -64,3 +65,130 @@ export function attachAttributesToLinks(formSchemaComponents) {
     }
   });
 }
+
+
+/**
+ * @function getAvailableDates
+ * Get All possible dates in given period with Term and Interval
+ *
+ * @param {Integer} keepAliveFor An integer for number of days, that tells form to be open for a particular period
+ * @param {String} keepAliveForInterval A string of days,Weeks,months, that tells form to be open for a particular period
+ * @param {Object[]} substartDate An object of Moment JS date, that tells form to be open on a particular day/date
```

Comment:
The documentation parameter does not match what the actual parameter is. Here it says `substartDate` but the actual parameter is `formStartDate`.